lms/envs/aws.py

```diff
@@ -133,6 +133,10 @@
 MEDIA_ROOT = ENV_TOKENS.get('MEDIA_ROOT', MEDIA_ROOT)
 MEDIA_URL = ENV_TOKENS.get('MEDIA_URL', MEDIA_URL)
 
+# GITHUB_REPO_ROOT is the base directory
+# for course data
+GITHUB_REPO_ROOT = ENV_TOKENS.get('GITHUB_REPO_ROOT', GITHUB_REPO_ROOT)
```

Comment:
Good question, I'm actually not sure what the difference is - they're both set to `ENV_ROOT / "data"` by default. I just moved it over because that's what the original api used:

```grep
./cms/djangoapps/contentstore/views/import_export.py:104:                data_root = path(settings.GITHUB_REPO_ROOT)
./cms/djangoapps/contentstore/views/import_export.py:270:                        settings.GITHUB_REPO_ROOT, [dirpath],
./cms/djangoapps/contentstore/views/tests/test_import_export.py:303:                settings.GITHUB_REPO_ROOT, [extract_dir / 'library'],
```